### PR TITLE
Update windows-autopilot-hybrid.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to reflect that the rule syntax needs to be edited (options have changed)

### CODE REVIEW

**Comment by PRMerger15** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ If you have a web proxy in your networking environment, ensure that the Intune C
 
 4. If you selected **Dynamic Devices** for the membership type, in the **Group** pane, select **Dynamic device members**.
 
-5. In the **Advanced rule** box, enter one of the following code lines:
+5. Click on **Edit** in the **Rule syntax** box, enter one of the following code lines:
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:155]:
```suggestion
5. Select **Edit** in the **Rule syntax** box and enter one of the following code lines:
```

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ If you have a web proxy in your networking environment, ensure that the Intune C
...
     - To create a group that includes all your Autopilot devices, enter `(device.devicePhysicalIDs -any _ -contains "[ZTDId]")`.
     - Intune's Group Tag field maps to the OrderID attribute on Azure AD devices. If you want to create a group that includes all of your Autopilot devices with a specific Group Tag(OrderID), type: `(device.devicePhysicalIds -any _ -eq "[OrderID]:179887111881")`
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:157]:
```suggestion
    - Intune's Group Tag field maps to the OrderID attribute on Azure AD devices. If you want to create a group that includes all of your Autopilot devices with a specific Group Tag (OrderID), type: `(device.devicePhysicalIds -any _ -eq "[OrderID]:179887111881")`.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dougeby Please merge this PR. Thanks

**Pull request merged**